bitacross-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -157,6 +162,37 @@ where
 		}
 	});
 
+	io.add_method("bitacross_checkSignBitcoin", move |_params: Params| {
+		debug!("worker_api_direct rpc was called: bitacross_checkSignBitcoin");
+		let request = prepare_check_sign_bitcoin_request(
+			signing_key_repository_cloned.as_ref(),
+			shielding_key_cloned_2.as_ref(),
+			ocall_api_cloned.as_ref(),
+		);
+		async move {
+			if let Ok(request) = request {
+				let params = Params::Array(vec![jsonrpc_core::Value::String(request.to_hex())]);
+				let json_value = match request_bit_across_inner(params).await {
+					Ok(value) => value.to_hex(),
```

Comment:
So we don't have a determinisitic result, we simply sign `[0u8; 32]` - and if it returns _some_ value it means OK?

Reply:
It can have 3 results:

1. Ok(true) - means ceremony **was sucesfully completed** (Ok) and signature verification **passed** (true)
2. Ok(false) - means ceremony **was sucesfully completed** (Ok) and signature verification **did not pass** (true) 
3. Err(error) - means ceremony **was not sucesfully** completed